src/components/SingleDatePicker/DatePicker.js

```diff
@@ -1,9 +1,14 @@
+/* eslint-disable no-unexpected-multiline */
```

Comment:
Its just needed to compile. The correct way should be remove it and change the code, but it should be made on new PR.

Reply:
This is removed in Anup's branch.

Reply:
Good! Here still need.

Reply:
yeah, but let's keep it here, to not through conflict with anup PR.